### DISCUSSION TITLE
Log forwarding

### DISCUSSION THREAD

**Discussion by Concieted** [telekom-security/tpotce#1014] [Q&A]
Good day 

could anyone kindly advise as to how I can go about forwarding Logs from my honeypot to my SIEM, please?

**Comment by Concieted** [comment]
I tried forwarding them by enabling Rsyslog 

then edited my rsyslog.conf file and added the log or collector I am attempting to forward logs too however that did not work. 

Please help

**Comment by Concieted** [comment]
the collectors IP rather

**Comment by t3chn0m4g3** [comment]
There have been numerous questions regarding log forwarding. All evolve around adjusting `logstash.conf` to your needs using `docker volumes` (see the Wiki and the links provided in #995).

**Comment by d3athg0d** [comment]
file_mode => 0760                                                                                                                                                                                                                        
if under smb means "Everyone" ?

path => "/data/suricata/log/suricata_ews.log"
can this be a SMB path like path => "\\1.2.3.4\data\log\suricata_ews.log" ?
